packages/director/src/app.ts

```diff
@@ -18,14 +18,14 @@ router
   .use(
     PROBE_LOGGER === 'true'
       ? expressPino({
-        logger: getLogger(),
-      })
+          logger: getLogger(),
```

Comment:
This is only a change because the settings from prettierrc.js were applied correctly.